automation/README.md

```diff
@@ -0,0 +1,35 @@
+# pulpcore RPMs
+
+This repository contains scripts and container files that can be used to add and update RPM packages for pulpcore to pulpcore-packaging.
+
+The container is designed to be run against an empty directory, and produce a set of commits against `pulpcore-packaging` containing the generated packages.
+
+## Dependencies
+
+* podman, docker, or some other container runtime
+
+## Configure
+
+For the tools to work properly, you'll need to provide the container with your name and email address. The simplest way is to create a `packaging.env` file like this:
+```
+NAME=John Doe
+EMAIL=john@example.org
+```
+
+The file can be later passed to the container runtime with `--env-file`. You can of course also use `--env` to specify the values of `NAME` and `EMAIL` directly.
+
+## Execution
+
+### Build
+
+```
+podman build --tag tfm-pulpcore-builder .
+```
+
+### Run
+
+1. create an empty directory to host the resulting packages
+2. pass the directory as a volume for `/app/packaging` to the container:
+```
+podman run --rm --volume $(pwd)/packaging/:/app/packaging/:Z  --env-file packaging.env tfm-pulpcore-builder
```

Comment:
I get SELinux denials when I share volumes. Is there some boolean I'm missing?

Reply:
Can you post them? The above works on my F32 with Selinux in enforcing when `$(pwd)/packaging/` is a newly created dir in my home

Reply:
Ah, I didn't see the `:Z`. Looks like that changes the file context:
```console
$ mkdir packaging
$ ls -lZd packaging/
drwxrwxr-x. 2 ekohl ekohl unconfined_u:object_r:user_home_t:s0 4096 Sep  4 10:55 packaging/
$ podman run --rm --volume $(pwd)/packaging/:/app/packaging/:Z -it centos:7
[root@b240c45dee80 /]# exit
$ ls -lZd packaging/
drwxrwxr-x. 2 ekohl ekohl system_u:object_r:container_file_t:s0:c123,c1003 4096 Sep  4 10:55 packaging/
```

Reply:
Yes, the :Z enforces it to be the right context for running the container.

Reply:
I learned something new today :)

Reply:
The best part of it: on systems without SELinux passing :Z (or :z) to --volume breaks Docker/podman, so you need to do funny hacks if you want scripts to work on both worlds:

https://github.com/Katello/katello-client-bootstrap/blob/master/Makefile#L5

Reply:
Would a little wrapper script that creates a default directory for me and just runs this command be asking too much?

Reply:
Absolutely!